packages/components/organisms/f-takeawaypay-activation/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.7.0
+------------------------------
+*August 9, 2021*
+
+### Added
+- Added logic to handle guest users
+- Code cleanup
+
+
 v0.6.0
```

Comment:
Should this be changed to v0.5.0 and the latest one will become v0.6.0?

Reply:
0.6.0 was merged to master already accidentally https://github.com/justeat/fozzie-components/pull/1173 . These packages are not used yet and were not published. Should I just change the readme and title of PR. but don't change the package.json?

Reply:
There will be a fozzie-bot warning that you need to bump the package version but we can ignore it in this case. I think changing the version not to miss one makes more sense because as you said package is not used yet

Reply:
@dandel10n Ok. Makes sense.